src/shared/firewall-util-nft.c

```diff
@@ -410,7 +416,6 @@ static int sd_nfnl_message_new_dnat_rule_out(sd_netlink *nfnl, sd_netlink_messag
                                              int family, const char *chain) {
         static const uint32_t zero, one = 1;
```

Comment:
Hmm, `zero` is not initialized.

Reply:
static storage duration gets auto-inited to 0. Do you want explicit initialisation?

Reply:
well, zero-initialization or not, the code would be more readable if you make this explicit and would generate the exact same compiled binary code. Since readability is important I'd thus initialize it here, to make this symmetric to the `one` case.